clients/p4rt_perf_test/p4rt_perf_test.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef P4RT_PERF_H
+#define P4RT_PERF_H
+
+#include <stdint.h>
+
+enum OPER { ADD = 1, DEL = 2, ADD_DEL = 3 };
+
+enum TEST_PROFILE { SIMPLE_L2_DEMO = 1 };
+
+struct ThreadInfo {
+  uint32_t tid;
+  uint32_t core_id;
+  uint64_t start;
+  uint64_t num_entries;
+  uint32_t oper;
+  double time_taken;
+};
+
+struct TestParams {
```

Comment:
Huh. I didn't know you could initialize structure members in a header file and include the file in more than one compilation unit.